litellm/integrations/langtrace.py

```diff
@@ -0,0 +1,143 @@
+import traceback
+import json
+from litellm.integrations.custom_logger import CustomLogger
+from litellm.proxy._types import SpanAttributes
+
+
+class LangtraceLogger(CustomLogger):
+    """
+    This class is used to log traces to Langtrace
+    """
+
+    def __init__(self) -> None:
+        try:
+            from langtrace_python_sdk import langtrace
+            from opentelemetry.trace import get_tracer
```

Comment:
@alizenhom we already have an OTEL Logger - https://github.com/BerriAI/litellm/blob/3007f0344d3a84c019377a27935bcc786bdaf253/litellm/integrations/opentelemetry.py#L41

If langtrace is otel compatible, the only thing we need is to show how to use it in docs - https://docs.litellm.ai/docs/proxy/logging#logging-proxy-inputoutput-in-opentelemetry-format

Reply:
I see, i will clean all this up 
1. use existing Otel Logger
2. will keep the extra file just to add extra attributes needed by langtrace
3. will adjust otel logger, if `callback_name` is `langtrace` then call `langtrace_set_attributes`